spec/system/admins_spec.rb

```diff
@@ -66,4 +66,24 @@
       expect(Admin.find_by(email: 'admin-other@example.com')).to be_blank
     end
   end
+
+  describe 'ユーザー削除' do
+    let!(:admin) { create(:admin) }
+
+    before do
+      create(:user, email: 'user@example.com')
+    end
+
+    it 'ユーザーを削除できる' do
+      login_as(admin, scope: :admin)
+      visit admins_users_path
+      expect do
+        click_button '削除', match: :first
+        accept_confirm '削除しますが、よろしいですか?'
```

Comment:
メッセージの内容が重要でないならこんな書き方もできます
```suggestion
        accept_confirm { click_button '削除', match: :first }
```

Reply:
いいですね。特にメッセージまでは重要でないので、修正しました
[45125ad](https://github.com/kk0000-kk/sakura-market/pull/22/commits/45125ad407b443dd713c7900df88879e0e0c3413)